public/js/app/components/AnalysisList/GridCell.tsx

```diff
@@ -0,0 +1,21 @@
+import { Col } from 'reactstrap'
+import ControlledPopover from '../ControlledPopover'
+import React from 'react'
+
+const GridCell: React.FC<{
+  id: string
+  header: string
+  content: React.ReactNode
+  popoverText?: string
+  md?: string
+}> = ({ id, header, content, popoverText, md = '4' }) => (
+  <Col md={md} className={`grid-cell`}>
+    <h5 className="cell-header">
```

Comment:
Okej! h5 is what is given in the sketches. Will check!

Reply:
Ah, just meant the semantic order of heading levels and not the style of it. Looks like the heading "before" in the DOM was an `<h3>`. But don't think we have been super strict about it before, so i don't think it absolutely must be fixed now.

Reply:
Yes! It was a bit of a trade off. I put a h3 instead of a h4 for the analysis name to keep the hierarchy clean, but left the h5 as is 😆 Thanks fot pointing it out!